starknet_py/net/full_node_client.py

```diff
@@ -93,6 +95,45 @@ async def get_block_traces(
     ) -> BlockTransactionTraces:
         raise NotImplementedError()
 
+    async def get_events(
+        self,
+        from_block_number: Optional[Union[int, Tag]] = None,
```

Comment:
Linter doesnt like that there are more than 5 params here, which was a result of matching the existing pattern of providing a block num and hash param.  Thoughts on whether to provide an exception or break into two methods, one accepting hash block ids and another number block ids?